lib/src/intl_suggestors/intl_migrator.dart

```diff
@@ -163,6 +170,20 @@ class IntlMigrator extends ComponentUsageMigrator {
     // }
   }
 
+  bool isLineIgnored(node) {
+    final ignoreComment = node.findPrevious(node.beginToken).precedingComments;
```

Comment:
nit: 
```
final ignoreComment = ....precedingComments ?? '';
return ignoreComment.value().contains(ignoreStatement);
```

Reply:
Oh, wow, I didn't know you could do that. I changed this whole thing, so this comment doesn't apply, but I will totally do that in this situation in the future.